text/0037-minting.md

```diff
@@ -0,0 +1,153 @@
+
+The contents of the `MintTx` include the full set of signatures of each of _M_ signers over the transaction's `TxPrefix`. 
+
+Each Consensus Validator node is configured with the approved signers list, and this list is also previously published to the chain via the `MintConfigTx`. When a Consensus Validator node sees a `MintTx` with an _M-of-N_ set of signatures, it considers the transaction valid, and consensus proceeds as normal. When a quorum is reached, and the nodes externalize the transaction, the minted TxOuts are written, along with a new block contents section, `mint_txs`, which contains:
+
+
+* The Token ID (only some Token Types allow minting, which is verified by the enclave)
+* The amount being minted
+* The destination's public subaddress view key
+* The destination's public subaddress spend key
+* the Ed25519 multisig signature set, including the total _M_ signatures required for the _M-of-N_ threshold, specified by the `MintConfigTx` previously accepted and written to the blockchain
+* Nonce: random value to protect against replay attacks
+* Tombstone block: block index after which this transaction will be discarded by consensus, to prevent the transaction from being nominated indefinitely
+
+
+When submitting a `MintTx`, we include a random nonce to protect against replay attacks, and a tombstone block to prevent the transaction from being nominated indefinitely, and these are committed to the chain.
+
+The block, and the latest `MintConfigTx`, ensure that the blockchain is therefore a self-contained document that can be audited to confirm that the minting was sound. In addition, minting is immune to replay attacks, meaning the same transaction cannot be submitted multiple times.
+
+### Privacy Properties of Minted Transaction Outputs
+
+Once the TxOuts are minted, they are exactly the same as any other TxOuts on chain, with the exception that the recipient view and spend public keys were published in the block's `MintTx`. The privacy guarantees for these TxOuts are that even with the view and spend published, one cannot verify with the TxOut alone that it was indeed sent to the specified public address. Rather, the public address is used by the enclaves to mint the amount requested to that address, and its publication in the block indicates that some number of the TxOuts in the block was sent to that address. The enclave does guarantee that at least one TxOut was sent to the specified public address. There is nothing stopping the block from containing other transactions for multiple token IDs, though in practice, it should be assumed that the TxOuts in a block with a `MintTx` should be considered "known" to have gone to that address, and all subsequent transactions from those TxOuts have full MobileCoin privacy guarantees.
```

Comment:
I don't think this is true. Since the minting has to be deterministic (multiple enclaves have to come up with identical TxOuts for any given set of input `MintTx`s), and since the `MintTx`s get published verbatim as part of the block, an outside party can simulate the code the enclave is running on those `MintTx`s and see what `TxOut`s they end up with.